### PR TITLE
test

### PR DESCRIPTION
**Pull request by LuisEulacioUCAB**

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -37,4 +37,8 @@ console.log('b', b);
 
 
 
+newCode("Natalia");
+newCode("Natalia");
```
Comment (lueucobuidlab) [inline test.js:41]: Remove this line

File: test.js [hunk 2]
```diff
@@ -37,4 +37,8 @@ console.log('b', b);
 
 
 
+newCode("Natalia");
```
Comment (lueucobuidlab) [inline test.js:40]: And this line

**Comment by LuisEulacioUCAB** [comment]
test

**Comment by LuisEulacioUCAB** [comment]
get 2

**Comment by lueucobuidlab** [comment]
dasdasdas

**Comment by LuisEulacioUCAB** [comment]
test

**Comment by LuisEulacioUCAB** [comment]
test 2 luis

**Comment by LuisEulacioUCAB** [comment]
testinggggg

**Pull request closed without merging**